src/data/finset.lean

```diff
@@ -78,6 +79,11 @@ theorem subset_def {s₁ s₂ : finset α} : s₁ ⊆ s₂ ↔ s₁.1 ⊆ s₂.1
 
 theorem subset.trans {s₁ s₂ s₃ : finset α} : s₁ ⊆ s₂ → s₂ ⊆ s₃ → s₁ ⊆ s₃ := subset.trans
 
+theorem superset.trans {s₁ s₂ s₃ : finset α} : s₁ ⊇ s₂ → s₂ ⊇ s₃ → s₁ ⊇ s₃ :=
+λ h' h, subset.trans h h'
+
+local attribute [trans] subset.trans superset.trans
```

Comment:
Why only `local`?

Reply:
It breaks some other modules otherwise. I didn't want to change them.

Reply:
Let's leave a `TODO: these should be global attributes, but this will require fixing other files`

Reply:
Do we want those rules available across mathlib. As I recall, it clashed with other rules. It might not be worth it.

Reply:
Well, they seem sensible and useful lemmas. Eventually we should either have them, or an explanation of why we can't have them (i.e. understand what they're clashing with). I don't think there's any reason that has to happen now, but a comment at least saying that we can't make them global would help others reading this later.

Reply:
Point taken